package.json

```diff
@@ -14,6 +14,7 @@
     "eslint",
     "linting",
     "react",
+    "react-native",
```

Comment:
Yes, but it's not a public npm module. We should add "private": true so one of us doesnt accidently publish it to npm